src/Kestrel/Kestrel.csproj

```diff
@@ -12,10 +12,13 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting" />
+    <PackageReference Include="Microsoft.AspNetCore.Certificates.Generation.Sources" PrivateAssets="All" />
+    <PackageReference Include="System.Security.Cryptography.Cng" />
```

Comment:
What's this for?

Reply:
He’s using the same code we have in tooling to locate the certificate (that’s why it’s a shared source library)

Reply:
We need that reference for the `RSACng` type.